common-lib/src/main/scala/com/gu/mediaservice/lib/ImageStorage.scala

```diff
@@ -13,6 +13,10 @@ import com.gu.mediaservice.model.MimeType
 object ImageStorageProps {
   val cacheDuration: Duration = 365 days
   val cacheForever: String = s"max-age=${cacheDuration.toSeconds}"
+  val filenameMetadataKey: String = "file_name"
```

Comment:
Does the latest version of localstack handle snake_case metadata properly now? Related https://github.com/guardian/grid/pull/2931.

Reply:
OMG 🤯 This is why we're not allowed nice things. Python shouldn't be allowed to be used by anyone older than 12.

Seriously though - also another example of relying on a simulator that isn't close enough to the real thing because it is by definition not the real thing. I guess the counter argument is that if we'd used it from the beginning then it would all be fine.

Thanks for pointing out.